banzai/qc/sinistro_1000s.py

```diff
@@ -36,13 +36,16 @@ def do_stage(self, images):
             logs.add_tag(logging_tags, 'FRAC1000', fraction_1000s)
             logs.add_tag(logging_tags, 'threshold', self.THOUSANDS_THRESHOLD)
             has_1000s_error = fraction_1000s > self.THOUSANDS_THRESHOLD
+            qc_results = {'sinistro_thousands.failed': has_1000s_error,
+                          'sinistro_thousands.fraction': fraction_1000s,
+                          'sinistro_thousands.threshold': self.THOUSANDS_THRESHOLD}
             if has_1000s_error:
                 self.logger.error('Image is mostly 1000s. Rejecting image', extra=logging_tags)
+                qc_results['rejected'] = True
```

Comment:
This ties into my question above about whether the failed flags should be there or not. Should we be consistent?

Reply:
We could have a default "rejected=false" flag, however in contrast to the "failed=false" flag I don't think it adds much information, and we would have to make sure to plumb it through properly (perhaps as an image attribute?)

Reply:
Ok. I think this is fine how it is.